aea/crypto/base.py

```diff
@@ -29,6 +29,7 @@
 from cryptography.hazmat.primitives import hashes, serialization
 from cryptography.hazmat.primitives.asymmetric import ec, utils
 from cryptography.hazmat.primitives.serialization import load_pem_private_key
+from .fetchai_base import FetchCrypto
```

Comment:
please provide full path for imports where possible, so `aea.crypto.fetchai_base`